releasenotes/notes/add-backend-v2-ce84c976fb13b038.yaml

```
@@ -0,0 +1,88 @@
+---
+features:
+  - |
+    Added a new version of the :class:`~qiskit.providers.Backend` interface,
+    :class:`~qiskit.providers.BackendV2`. This new version is a large change
+    from the previous version, :class:`~qiskit.providers.BackendV1` and
+    changes both the user access pattern for properties of the backend (like
+    number of qubits, etc) and how the backend represents its constraints
+    to the transpiler. The execution of circuits (via the
+    :meth:`~qiskit.providers.BackendV2.run` method) remains unchanged. With
+    a :class:`~qiskit.providers.BackendV2` backend instead of having a separate
+    :meth:`~qiskit.providers.BackendV1.configuration`,
+    :meth:`~qiskit.providers.BackendV1.properties`, and
+    :meth:`~qiskit.providers.BackendV1.defaults` methods that construct
+    :class:`~qiskit.providers.models.BackendConfiguration`, 
+    :class:`~qiskit.providers.models.BackendProperties`, and
+    :class:`~qiskit.providers.models.PulseDefaults` objects respectively,
+    like in the :class:`~qiskit.providers.BackendV1` interface, the attributes
+    contained in those output objects are accessible directly as attributes of
+    the :class:`~qiskit.providers.BackendV2` object. For example, to get the
+    number of qubits for a backend with :class:`~qiskit.providers.BackendV1`
+    you would do::
+
+      num_qubits = backend.configuration().n_qubits
+
+    while with :class:`~qiskit.providers.BackendV2` it is::
+
+      num_qubits = backend.num_qubits
```

Needs a four-space indent.

```suggestion
    you would do::

        num_qubits = backend.configuration().n_qubits

    while with :class:`~qiskit.providers.BackendV2` it is::

        num_qubits = backend.num_qubits
```

---

I think it still renders fine like this as long as it's partially indented but this is definitely clearer.

---

Oh, I must have done it wrong in the past, then - I've definitely had problems in the release notes because of the two-space indent.  Perhaps it's when you try to use `.. jupyter-execute` instead.